assets/scripts/info_bubble/InfoBubble.jsx

```diff
@@ -204,7 +235,7 @@ class InfoBubble extends React.Component {
     let position
     switch (type) {
       case INFO_BUBBLE_TYPE_SEGMENT:
-        position = this.props.dataNo
+        position = window.parseInt(this.props.dataNo)
```

Comment:
Oh no, you did do that originally in the action creator, but since we are now passing in 'left' or 'right' as the `dataNo` when the segment hovered is a building, it causes the saved value to be a `null` or `undefined`, I forgot which one which doesn't help when determining the type. 

I guess a solution would be if the `dataNo` can be parsed as a int then save it as an int and if not save it as a string?